css/slider.css

```diff
@@ -0,0 +1,204 @@
+.slider-main{
```

Comment:
add Prettier plugin for your IDE and create hotkey for it. use it every time when you save the file.
between selector and brace should be space.